docs/mathematics/Matrix-Addition.mdx

```diff
@@ -0,0 +1,177 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+    int r1, c1, r2, c2;
+    cout << "Enter the number of rows and columns of the first matrix:  ";
+    cin >> r1 >> c1;
+    cout << "Enter the number of rows and columns of the second matrix: ";
+    cin >> r2 >> c2;
+
+    //If the given matrices differ in thier number of rows and columns, they cannot be added
+    if ((r1 != r2) || (c1 != c2))
+    {
+        cout << "Given Matrices cannot be added!!!";
+        return 0;
+    }
+    int A[r1][c1], B[r2][c2], C[r1][c1] = {0};
+
+    // Input the values of the matrices
+    cout << "Enter the values of the first matrix\n";
+    for (int i = 0; i < r1; i++)
+    {
+        for (int j = 0; j < c1; j++)
+            cin >> A[i][j];
+    }
+    cout << "Enter the values of the second matrix\n";
+    for (int i = 0; i < r2; i++)
+    {
+        for (int j = 0; j < c2; j++)
+            cin >> B[i][j];
+    }
+
+    for (int i = 0; i < r1; i++)
+    {
+        for (int j = 0; j < c2; j++)
+        {
+            // Add the corresponding values of both the matrices
+            C[i][j] = A[i][j] + B[i][j];
+        }
+    }
+
+    cout << "The resultant matrix is:\n";
+    for (int i = 0; i < r2; i++)
+    {
+        for (int j = 0; j < c2; j++)
+            cout << C[i][j] << " ";
+        cout << endl;
+    }
+
+    return 0;
+}
+```
+
+</TabItem>
+<TabItem value="py">
+
+```python
+Row_1 , Col_1 = map(int,input("Enter the number of rows and columns of the first matrix:  ").split(" "))
+Row_2 , Col_2 = map(int,input("Enter the number of rows and columns of the second matrix:  ").split(" "))
+
+if(Row_1 != Row_2 or Col_1 != Col_2):
+    print( "Given Matrices cannot be added!!!")
+    quit()
+
+matrix_1 = []
+print("Enter the values of the first matrix")
+for i in range(Row_1):
+    matrix_1.append([int(x) for x in input().split(" ")])
+
+matrix_2 = []
+print("Enter the values of the second matrix")
+for i in range(Row_2):
+    matrix_2.append([int(x) for x in input().split(" ")])
+
+result = [[0 for i in range(Col_2)] for j in range(Row_1)]
+# Adding both matrices and storing in result
+for i in range(Row_1):
+    for j in range(Col_2):
+        result[i][j] = matrix_1[i][j] + matrix_2[i][j]
+
+print("The resultant matrix is:")
+for i in range(Row_1):
+    for j in range(Col_2):
+        print(result[i][j], end=" ")
+    print()
+    
+```
+
+</TabItem>
+</Tabs>
+
+## Sample Input and Output
+
+### Input
+
+```
+Enter the number of rows and columns of the first matrix:  3 3
+Enter the number of rows and columns of the second matrix:  3 3
+Enter the values of the first matrix
+1 2 4
+5 3 6
+7 5 8
+Enter the values of the second matrix
+4 5 6
+8 6 4
+2 4 6
+```
+
+### Output
+
+```
+The resultant matrix is:
+5 7 10
+13 9 10
+9 9 14
+```
+
+## Complexity Analysis
+
+```
+Time complexity - O(n * m)
+Space complexity - O(n*m),, where 'n' and 'm' are the number of rows and columns respectively
+```
```

Comment:
Don't keep them in a triple backticks. Keep them as a bullet-point, and add backticks on the values and around `n` and `m`